content/docs/reference/tutorials/aws/tutorial-rest-api.md

```diff
@@ -146,4 +146,3 @@ From here, you can dive deeper:
         REST APIs that use a document database
   - [Multi-cloud Serverless plus Containers]({{< relref "/docs/quickstart/cloudfx/tutorial-thumbnailer.md" >}}): Create a multi-cloud video
```

Comment:
```suggestion
  - [Multi-cloud Serverless plus Containers]({{< relref "/docs/quickstart/cloudfx/tutorial-thumbnailer" >}}): Create a multi-cloud video
```

When I sat down with @chrsmith before, we started removing `md` in the `relref` links. Also, is my understanding correct that having a trailing slash for these breaks the build? As in, this would trigger a failed build: `relref "/docs/quickstart/cloudfx/tutorial-thumbnailer/"`

Reply:
According to https://gohugo.io/content-management/cross-references/#use-ref-and-relref:

> The single parameter to `ref` is a string with a content `documentname` (e.g., `about.md`) with or without an appended in-document `anchor` (`#who`) without spaces. Hugo is flexible in how we search for documents, so the file suffix may be omitted.

That last sentence: 

> Hugo is flexible in how we search for documents, **so the file suffix may be omitted**.

So it does indeed look like we could remove the `.md` suffix. But I guess adding a trailing `/` will make it so Hugo can't find the content page and fail the build.